src/components/chip-set/chip-set.tsx

```diff
@@ -273,38 +260,33 @@ export class ChipSet {
                         class={`mdc-text-field__input ${
                             hiddenInput ? 'hidden' : ''
                         }`}
-                        value={textValue}
+                        value={this.textValue}
                         onBlur={this.handleInputBlur}
                         onFocus={this.handleTextFieldFocus}
                         onInput={this.handleTextInput}
                         // Some browsers emit a change event on input elements, we need to stop
                         // that event from propagating since we are emitting our own change event
                         onChange={this.inputFieldOnChange}
+                        placeholder={this.searchLabel}
                     />
                 </div>
                 <label
                     class={{
                         'mdc-floating-label': true,
-                        'mdc-floating-label--float-above': !!(
-                            textValue || this.editMode
-                        ),
                         'mdc-text-field--disabled': this.disabled,
                         'mdc-text-field--required': this.required,
+                        'force-float': !!(this.value.length || this.editMode),
```

Comment:
This works because `MDCTextField` doesn't know about this class, and doesn't try to add or remove it.